src/mlpack/methods/nmf/nmf_main.cpp

```diff
@@ -153,10 +213,7 @@ static void mlpackMain()
       true, "max_iterations must be non-negative");
 
   RequireAtLeastOnePassed({ "h", "w" }, false, "no output will be saved");
-  RequireNoneOrAllPassed({"initial_w", "initial_h"}, true);
 
-  // Load input dataset.  We know if the data is transposed based on the
-  // BINDING_MATRIX_TRANSPOSED macro, which will be 'true' or 'false'.
```

Comment:
Was there a need to remove this comment?

Reply:
I did not remove this comment. Instead I move it into the function void ApplyFactorization() at Line 144 and 145 :)

Reply:
Ah, got it, thanks. Sorry that I missed it. :+1: